js/defaults.js

```diff
@@ -8,7 +8,7 @@
  */
 
 var defaults = {
-	port: 8080,
+	port: process.env.MM_PORT || 8080,
```

Comment:
@MichMich I did run grunt before and couldn't figure out what the problem was. jsLint says there are irregular whitespaces but there aren't any. Or am I completely wrong here?